llvm/include/llvm/MC/MCContext.h

```diff
@@ -599,8 +600,18 @@ class MCContext {
                                                    unsigned Flags,
                                                    unsigned EntrySize);
 
-  MCSectionGOFF *getGOFFSection(StringRef Section, SectionKind Kind,
-                                MCSection *Parent, uint32_t Subsection = 0);
+private:
```

Comment:
instead of adding a `private`, just move the `getGOFFSection` to the existing private part.

Reply:
Moved the code.